Packages/com.unity.inputsystem/InputSystem/State/InputStateBlock.cs

```diff
@@ -42,89 +42,89 @@ public unsafe struct InputStateBlock
         public const uint AutomaticOffset = 0xfffffffe;
 
         // Primitive state type codes.
-        public static readonly FourCC kTypeBit = new FourCC('B', 'I', 'T');
-        public static readonly FourCC kTypeSBit = new FourCC('S', 'B', 'I', 'T');
-        public static readonly FourCC kTypeInt = new FourCC('I', 'N', 'T');
-        public static readonly FourCC kTypeUInt = new FourCC('U', 'I', 'N', 'T');
-        public static readonly FourCC kTypeShort = new FourCC('S', 'H', 'R', 'T');
-        public static readonly FourCC kTypeUShort = new FourCC('U', 'S', 'H', 'T');
-        public static readonly FourCC kTypeByte = new FourCC('B', 'Y', 'T', 'E');
-        public static readonly FourCC kTypeSByte = new FourCC('S', 'B', 'Y', 'T');
-        public static readonly FourCC kTypeLong = new FourCC('L', 'N', 'G');
-        public static readonly FourCC kTypeULong = new FourCC('U', 'L', 'N', 'G');
-        public static readonly FourCC kTypeFloat = new FourCC('F', 'L', 'T');
-        public static readonly FourCC kTypeDouble = new FourCC('D', 'B', 'L');
+        public static readonly FourCC TypeBit = new FourCC('B', 'I', 'T');
+        public static readonly FourCC TypeSBit = new FourCC('S', 'B', 'I', 'T');
+        public static readonly FourCC TypeInt = new FourCC('I', 'N', 'T');
+        public static readonly FourCC TypeUInt = new FourCC('U', 'I', 'N', 'T');
+        public static readonly FourCC TypeShort = new FourCC('S', 'H', 'R', 'T');
+        public static readonly FourCC TypeUShort = new FourCC('U', 'S', 'H', 'T');
+        public static readonly FourCC TypeByte = new FourCC('B', 'Y', 'T', 'E');
+        public static readonly FourCC TypeSByte = new FourCC('S', 'B', 'Y', 'T');
+        public static readonly FourCC TypeLong = new FourCC('L', 'N', 'G');
+        public static readonly FourCC TypeULong = new FourCC('U', 'L', 'N', 'G');
+        public static readonly FourCC TypeFloat = new FourCC('F', 'L', 'T');
+        public static readonly FourCC TypeDouble = new FourCC('D', 'B', 'L');
```

Comment:
Looking at the use of these, I think `Type` was a misnomer here. They are format identifiers, not type identifiers. The respective API in `IInputStateTypeInformat` is `GetFormat` and the respective InputStateBlock field is `format`. Should we rename all of these to be `FormatBit` etc?

Reply:
Makes sense. Fixed.